gnuplot/examples/inverse_api.rs

```diff
@@ -77,14 +77,6 @@ fn lines<'l, Tx: IntoIterator<Item = f32>, Ty: IntoIterator<Item = f32>>(x: Tx,
 	}
 }
 
-impl Lines
```

Comment:
Please revert this. I guess it's not used, but the presense of this code is important to convey the idea of this API.

Reply:
Reverted, added `#[allow(dead_code)]` to the Impl